c/tskit/tables.c

```diff
@@ -771,12 +903,16 @@ tsk_individual_table_load(tsk_individual_table_t *self, kastore_t *store)
         ret = TSK_ERR_BAD_OFFSET;
         goto out;
     }
+    if (parents_offset[num_rows] != parents_length) {
+        ret = TSK_ERR_BAD_OFFSET;
```

Comment:
We should try to cover this line. @benjeffery, where do we catch this sort of stuff again?

Reply:
This sort of thing should be hit by adding the column to the list of ragged columns in test_tables.py:845. I'm not sure why it isn't, but could look into it.

Reply:
I've added that at test_tables.py:847, in addition to some basic tests below. I was also confused why the codecov/patch did not pass.